libs/expo/betterangels/src/lib/ui-components/ClientCard.tsx

```diff
@@ -96,16 +96,16 @@ export default function ClientCard(props: IClientCardProps) {
       ]}
     >
       <Avatar
-        accessibilityHint={`shows avatar of ${client.user.firstName} ${client.user.lastName} if available`}
-        accessibilityLabel={`Avatar of ${client.user.firstName} ${client.user.lastName} client`}
+        accessibilityHint={`shows avatar of ${client.firstName} ${client.lastName} if available`}
+        accessibilityLabel={`Avatar of ${client.firstName} ${client.lastName} client`}
         imageUrl={client.profilePhoto?.url}
         size="xl"
         mr="xs"
       />
 
       <View style={{ gap: Spacings.xxs, flex: 2 }}>
         <TextBold size="sm">
-          {client.user.firstName} {client.user.lastName}{' '}
+          {client.firstName} {client.lastName}{' '}
```

Comment:
this `client` is `ClientProfileQuery`; it should be `ClientProfileType`. This needs to be fixed in too many places to address in this ticket. Made a [debt ticket](https://betterangels.atlassian.net/browse/DEV-1678)